tests/qt_types_standalone/cpp/qfontmetrics.h

```diff
@@ -0,0 +1,59 @@
+// clang-format off
+// SPDX-FileCopyrightText: 2024 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+// clang-format on
+// SPDX-FileContributor: Laurent Montel <laurent.montel@kdab.com>
+//
+// SPDX-License-Identifier: MIT OR Apache-2.0
+#pragma once
+
+#include <QtGui/QFont>
+#include <QtGui/QFontMetrics>
+#include <QtGui/QGuiApplication>
+#include <QtTest/QTest>
+
+#include "cxx-qt-gen/qfontmetrics.cxx.h"
+
+class QFontMetricsTest : public QObject
+{
+  Q_OBJECT
+
+private Q_SLOTS:
+  void construct()
+  {
+    std::vector<char*> args;
+    std::string path = "/path";
+    args.push_back(path.data());
+    auto argc = static_cast<int>(args.size());
+
+    // QFontDatabase needs a QGuiApplication first
+    QGuiApplication app(argc, args.data());
+    app.setApplicationName(QStringLiteral("kdab"));
+
+    QFont f;
+    const int pointSize = 40;
+    f.setPointSize(pointSize);
+    const auto m = constructor_qfontmetrics(f);
+    QCOMPARE(m.ascent(), 57);
```

Comment:
value depend against font size etc...
=> we can't test it...

No idea what testing from constructor...